fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRUserTransactionAdapter.java

```diff
@@ -89,6 +93,14 @@ public void begin() throws FHIRPersistenceException {
             // the transaction is active but has already been marked for rollback. This is OK,
             // we just behave as though we're a nested transaction
             this.startCount++;
+        } else if (isActive(status) && bulk) {
```

Comment:
Might be worth adding a new test to the transaction test class to make sure things work OK when bulk is set on the request context.

Reply:
Added test